test/service.ts

```diff
@@ -1864,10 +1938,12 @@ describe('GrpcService', () => {
   });
 
   describe('ObjectToStructConverter', () => {
-    let objectToStructConverter;
+    // tslint:disable-next-line no-any
+    let objectToStructConverter: any;
```

Comment:
I'm guessing setting this to `ObjectToStructConverter` causes a sea of errors, right?

Reply:
If the approach with `GrpcService: GrpcService` works out, maybe let's do the same with this one